idr/__init__.py

```diff
@@ -1,22 +1,22 @@
-from .connections import connection # NOQA
-from .connections import create_http_session # NOQA
+from .connections import connection  # NOQA
+from .connections import create_http_session  # NOQA
 
-from .attributes import attributes_by_attributes # NOQA
-from .attributes import annotation_ids_by_field # NOQA
-from .attributes import get_phenotypes_for_gene # NOQA
-from .attributes import get_phenotypes_for_genelist # NOQA
-from .attributes import get_similar_genes # NOQA
-from .attributes import get_organism_screenids # NOQA
+from .attributes import attributes_by_attributes  # NOQA
+from .attributes import annotation_ids_by_field  # NOQA
+from .attributes import get_phenotypes_for_gene  # NOQA
+from .attributes import get_phenotypes_for_genelist  # NOQA
+from .attributes import get_similar_genes  # NOQA
+from .attributes import get_organism_screenids  # NOQA
 
-from .images import images_by_phenotype # NOQA
-from .images import simple_colocalisation # NOQA
+from .images import images_by_phenotype  # NOQA
+from .images import simple_colocalisation  # NOQA
 
-from .externalDBs import genes_of_interest_from_string # NOQA
-from .externalDBs import network_of_interest # NOQA
-from .externalDBs import get_entrezid # NOQA
-from .externalDBs import get_ensembleid # NOQA
-from .externalDBs import genes_of_interest_go # NOQA
-from .externalDBs import ensembleid_to_genesymbol # NOQA
+from .externalDBs import genes_of_interest_from_string  # NOQA
+from .externalDBs import network_of_interest  # NOQA
+from .externalDBs import get_entrezid  # NOQA
+from .externalDBs import get_ensembleid  # NOQA
+from .externalDBs import genes_of_interest_go  # NOQA
+from .externalDBs import ensembleid_to_genesymbol  # NOQA
 
-from .visualizations import plot_idr_attributes # NOQA
-from .visualizations import plot_string_interactions # NOQA
+from .visualizations import plot_idr_attributes  # NOQA
+from .visualizations import plot_string_interactions  # NOQA
```

Comment:
FYI you can avoid using `# NOQA` by adding the imports to `__all__`, see for example
https://github.com/ome/omero-rois/blob/28220e0ee7a5a47f9768e0a64cdf0637770aa4dc/src/omero_rois/__init__.py#L20-L32

Not a blocker to merging, unless someone else thinks otherwise.